locales-src/newtab.wip

```diff
@@ -0,0 +1,10 @@
+newtab-page-title = New Tab
+
+newtab-search-box-search-text = Search
+newtab-search-box-search-button =
+  .title = { newtab-search-box-search-text }
+
+newtab-search-box-search-the-web-text = Search the Web
+newtab-search-box-search-the-web-input =
+  .placeholder = { newtab-search-box-search-the-web-text }
+  .title = { newtab-search-box-search-the-web-text }
```

Comment:
One thing is to do 

```
newtab-search-box-search-the-web-input =
  .placeholder = Search the Web
  .title = { newtab-search-box-search-the-web-input.placeholder }
```

Or referencing a button label from another message. This creates way too many cross-references.

If you ever end up changing the first string, we'll need to invalidate all the strings referencing it, and that might not be wanted or needed. Also, it doesn't help with translation memories or typing less.

Reply:
So is the suggestion to just do:
```
newtab-search-box-search-the-web-text = Search the Web
newtab-search-box-search-the-web-input =
  .placeholder = Search the Web
  .title = Search the Web
```

Or you're saying only use references among attributes of the same parent?

Reply:
Either this, or just reference within the same message. Both work for me

Reply:
Removed the references